audio/kmix/Makefile

```diff
@@ -10,19 +10,13 @@ LICENSE=	GPLv2
 
 LIB_DEPENDS=	libasound.so:audio/alsa-lib
 
-USES=		cmake compiler:c++11-lang gettext kde:5 qt:5 tar:xz xorg
+USES=		cmake compiler:c++11-lang gettext kde:6 qt:6 tar:xz xorg
 USE_KDE=	auth codecs config configwidgets coreaddons dbusaddons \
-		globalaccel i18n iconthemes init itemmodels itemviews kcmutils \
-		notifications package plasma-framework service solid \
-		widgetsaddons windowsystem xmlgui \
-		ecm:build
-USE_QT=		core dbus gui widgets xml \
-		buildtools:build qmake:build
+		globalaccel i18n iconthemes itemmodels itemviews kcmutils \
+		notifications package statusnotifieritem service solid widgetsaddons windowsystem xmlgui ecm:build
```

Comment:
This line looks long. Please keep within 80 characters and keep ecm:build on a separate line.